hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/web/ozShell/token/GetTokenHandler.java

```diff
@@ -39,7 +40,9 @@
     description = "get a delegation token.")
 public class GetTokenHandler extends Handler {
 
-
+  @CommandLine.Parameters(arity = "1..1",
+      description = Shell.OZONE_BUCKET_URI_DESCRIPTION)
```

Comment:
OZONE_BUCKET_URI_DESCRIPTION -> need to be updated as o3://om or o3://<<serviceid>>

Reply:
Fixed.